cpp/src/parquet/types.cc

```diff
@@ -34,32 +34,44 @@ using arrow::util::Codec;
 
 namespace parquet {
 
-std::unique_ptr<Codec> GetCodecFromArrow(Compression::type codec) {
-  std::unique_ptr<Codec> result;
+namespace {
+arrow::Compression::type TranslateParquetCompressionTypeToArrowCompressionType(
```

Comment:
Seems silly to me to continue to maintain this code. Can we do `using Compression = ::arrow::Compression` in `parquet/types.h` and move on?

Reply:
By this, you mean parquet::Compression to be removed and replaced with arrow::Compression?
The API would be broken then since this would also expose Compression::BZ2 which is not a valid compression algorithm for Parquet.
I do not know the history of the project and why this is so.

Reply:
I think we would prohibit unsupported compressions in Parquet. 

@xhochy @majetideepak any thoughts about this?

Reply:
I'm fine with this either way as long as we are to get programmatically which codecs are supported by Parquet.